src/marqo/s2_inference/multimodal_model_load.py

```diff
@@ -126,7 +126,7 @@ def __init__(self, model):
         self.model = model
 
     def encode(self, content, modality, **kwargs):
-        return self.model.encode(content)
+        return self.model.encode(content, **kwargs)
```

Comment:
See https://marqo-ai.slack.com/archives/C05K6BFQUTF/p1728357871797459 for details about this bugfix